gulpfile.js

```diff
@@ -132,52 +140,48 @@ gulp.task('build:npm:package.json', function (done) {
 
   mkdirp.sync(NPM_PATH);
 
-  fs.writeFile(NPM_PATH + '/' + 'package.json', JSON.stringify(pkg, null, 2), done);
-});
+  fs.writeFile(path.join(NPM_PATH, 'package.json'), JSON.stringify(pkg, null, 2), done);
+}
 
-gulp.task('build:npm:src', function () {
-  return gulp.src([
-    'src/**/*.js'
-  ])
-  .pipe(replace('@DOT_MIN', ''))
-  .pipe(envify(process.env))
-  .pipe(brfs())
-  .pipe(gulp.dest(NPM_PATH));
-});
+function npmSrc() {
+  return src('src/**/*.js')
+    .pipe(replace('@DOT_MIN', ''))
+    .pipe(brfs())
+    .pipe(dest(NPM_PATH));
+}
 
-gulp.task('build:npm:browser', function () {
+function npmBrowser() {
   var browserPath = NPM_PATH + '/dist/browser/';
 
   mkdirp.sync(browserPath);
 
-  return gulp.src([
-    'dist/web/dropin/' + VERSION + '/js/dropin.js'
-  ])
-  .pipe(gulp.dest(browserPath));
-});
-
-gulp.task('build:npm', [
-  'build:npm:css',
-  'build:npm:statics',
-  'build:npm:package.json',
-  'build:npm:src',
-  'build:npm:browser'
-]);
-
-gulp.task('clean', function () {
-  return del(['./dist']);
-});
+  return src(path.join(
+    'dist/web/dropin', VERSION, 'js/dropin.js'
+  ))
+    .pipe(dest(browserPath));
+}
 
-gulp.task('build', function (done) {
-  process.env.npm_package_version = VERSION;
```

Comment:
This is the only place i could find where the env variable was being set (not some external .env file, nor did it have logic to defer to an external env file). Note that `VERSION` is set by `require('package.json').version` on L27 and has been since initial commit when this was open-sourced.